spec/dummy/spec/requests/users/registration_requests_spec.rb

```diff
@@ -31,6 +31,16 @@
         expect(response.headers['Expire-At']).to be_present
         expect(response.headers['Refresh-Token']).to be_present
       end
+
+      it 'should return 422 - email has already been taken' do
```

Comment:
Can you move this to invalid params context?

Reply:
I thought about putting it there, but it's not technically invalid params. The params are valid, it's just a validation error.

Reply:
Got it. I have created the invalid params context for grouping all invalid cases. Maybe we need to change the context name, I will merge this now and change the context name later. Thanks.